shared/message/input/Image.proto

```diff
@@ -25,85 +25,34 @@ import "google/protobuf/timestamp.proto";
 import "Matrix.proto";
 import "Vector.proto";
 
-/**
- * Camera, lens, and output image information.
- */
 message Image {
 
-    /**
-     * The lens type and details about the images it makes.
-     */
     message Lens {
-
-        /**
-         * Types of projection of our lenses.
-         */
         enum Projection {
             UNKNOWN     = 0;
             RECTILINEAR = 1;
             EQUIDISTANT = 2;
             EQUISOLID   = 3;
         }
-
-        /**
-         * The type of projection this lens is using.
-         */
         Projection projection = 1;
-
-        /**
-         * Normalised focal length: focal length in pixels / image width.
-         */
+        /// Normalised focal length: focal length in pixels / image width
```

Comment:
@TrentHouliston are these still normalised? I thought we stopped using normalised coordinates? Or was that somewhere else?

Reply:
@TrentHouliston